lib/packaging/util/net.rb

```
@@ -362,9 +362,9 @@ def remote_bootstrap(host, treeish, tar_cmd = nil, tarball = nil)
 bundle_prefix= ;
 if [[ -r Gemfile ]]; then
   source /usr/local/rvm/scripts/rvm; rvm use ruby-2.4.1; bundle install --path .bundle/gems ;
-  bundle_prefix='bundle exec' ;
+else
+  echo "ERROR: Couldn't read Gemfile, can't bundle install."
```

I think this is going to break ezbake projects until things are cleared up with ezbake using packaging as a gem / all ezbake projects are updated.

Also probably want to actually raise an error in this case, not let the build continue

---

oops, forgot the `exit 1` I was gonna put in there.

and yeah, this will break ezbake projects at the moment, but I figure we can either sit on merging this or just not do a packaging release 'til everything is updated. does that seem doable?

---

not doing a packaging release won't help here since ezbake builds just pull from 1.0.x

---

right, until ezbake gets updated.